Nos/Views/ProfileEdit/ProfileEditView.swift

```diff
@@ -40,10 +47,45 @@ struct ProfileEditView: View {
     }
 
     var body: some View {
+        let avatarSize: CGFloat = 99
+        
         NosForm {
-            AvatarView(imageUrl: URL(string: avatarText), size: 99)
+            ImagePickerButton(cameraDevice: .front) { imageURL in
```

Comment:
I think we need to disable video upload in this case, which also means we need to change the text on the "Take a photo or video" alert action in this flow. 😬

Reply:
Done.